src/Controls/StreetControls.js

```diff
@@ -0,0 +1,111 @@
+import FirstPersonControls from 'Controls/FirstPersonControls';
+import AnimationPlayer, { AnimatedExpression } from 'Core/AnimationPlayer';
```

Comment:
`AnimationPlayer` and `AnimatedExpression` will be removed soon because it's better to use TWEEN.js. This to be done later.